contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
```

Comment:
> it should automatically "leaveGame" by the contract.
Why would you automatically leave? By making a case against the service provider about the service not being provided is not the same as wishing to leave.